### PR TITLE
Flow collector on podman sites

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: cmd/skupper/skupper_podman_site.go [hunk 1]
```diff
@@ -50,13 +51,19 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
...
+		IngressHosts:                 s.flags.IngressHosts,
+		IngressBindIPs:               s.flags.IngressBindIPs,
+		IngressBindInterRouterPort:   s.flags.IngressBindInterRouterPort,
+		IngressBindEdgePort:          s.flags.IngressBindEdgePort,
+		IngressBindFlowCollectorPort: s.flags.IngressBindFlowCollectorPort,
+		ContainerNetwork:             s.flags.ContainerNetwork,
+		PodmanEndpoint:               s.flags.PodmanEndpoint,
+		EnableFlowCollector:          routerCreateOpts.EnableFlowCollector,
+		EnableConsole:                routerCreateOpts.EnableConsole,
+		ConsoleUser:                  routerCreateOpts.User,
```
Comment (ajssmith) [inline cmd/skupper/skupper_podman_site.go:63]: Suggest adding console auth to allow for unsecured configuration if desired

File: pkg/domain/podman/site.go [hunk 2]
```diff
@@ -458,6 +489,18 @@ func (s *SiteHandler) Get() (domain.Site, error) {
 			case *domain.Router:
 				site.RouterOpts.DebugMode = c.Env["QDROUTERD_DEBUG"]
 				site.RouterOpts.Logging = qdr.GetRouterLogging(config)
+			case *domain.FlowCollector:
+				enableConsole, _ := strconv.ParseBool(c.Env["ENABLE_CONSOLE"])
+				site.EnableConsole = enableConsole
+				site.EnableFlowCollector = true
+				site.FlowCollectorRecordTtl, _ = time.ParseDuration(c.Env["FLOW_RECORD_TTL"])
+				site.IngressBindFlowCollectorPort = c.SiteIngresses[0].GetPort()
```
[the remark this answers is not in the document]
Comment (fgiorgetti) [reply pkg/domain/podman/site.go:497]: Yes, if flow-collector is enabled, the host port must be bound, regardless of the ingress mode chosen.
Fixing it.

File: cmd/skupper/skupper_podman_site.go (hunk 1 above)
Comment (fgiorgetti) [reply cmd/skupper/skupper_podman_site.go:64]: Done.

**Comment by fgiorgetti** [comment]
@ajssmith just added tests to validate flow collector on podman sites.

**Review by grs — APPROVED** [review]

**Pull request merged**